device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/pai.profile

```diff
@@ -0,0 +1 @@
+SAI_KEY_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/gearbox_100G_PAM4.xml
```

Comment:
can you rename this file to psai.profile?

Reply:
Yes